src/components/ApeTextField/ApeTextField.tsx

```diff
@@ -71,7 +71,6 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(0, 1),
   },
   helperBox: {
-    height: 18,
```

Comment:
I had done this to get the create circle page to have it so when errors appeared, they didn't reflow the Ui, so if this is wierd in other places, I'd rather use the fixed height everywhere and adjust the spacing on the grid. We don't really have a unified standard for these types of things, something needing to work with our designer on.

Reply:
Also these can all switch to the inline errors, which should work except for epoch has really long errors..